packages/blade/src/components/Amount/Amount.tsx

```diff
@@ -19,6 +19,17 @@ import type { StyledPropsBlade } from '~components/Box/styledProps';
 
 type Currency = 'INR' | 'MYR';
 
+type SizeType =
```

Comment:
What I meant was can we define the types directly rather than interpolating as template literal types? Because the sizes are limited it can simplify readability, i.e. `type SizeType = 'title-small' | 'title-medium' | ...`

Reply:
https://github.com/razorpay/blade/pull/1010#discussion_r1136922692

Reply:
Okay, let's go ahead for now as it's.